defaults/main.yml

```diff
@@ -1,7 +1,9 @@
 ---
 # Install from phar (faster, but less flexible).
 drush_version: "8.1.10"
-drush_phar_url: https://github.com/drush-ops/drush/releases/download/{{ drush_version }}/drush.phar
+# Install Drush Launcher (Drupal 8.4+).
+drush_launcher_version: "0.3.1"
+drush_phar_url: https://github.com/drush-ops/drush-launcher/releases/download/{{ drush_launcher_version }}/drush.phar
```

Comment:
Since the [Drush 8.x docs](http://docs.drush.org/en/8.x/install/) still instruct to download the Drush phar instead of `drush-launcher`, I suggest using the main Drush phar instead of `drush-launcher` for drush <= `9`